okio/src/jvmTest/kotlin/okio/FileLeakTest.kt

```diff
@@ -109,3 +145,18 @@ private inline fun <R> ZipOutputStream.putEntry(name: String, action: BufferedSi
     closeEntry()
   }
 }
+
+// This is a Linux only test for open file descriptors on the current process
+@OptIn(ExperimentalPathApi::class)
+private fun Path.assetFileNotOpen() {
+  val fds = Path("/proc/self/fd")
+  if (fds.isDirectory()) {
+    // Linux: verify that path is not open
```

Comment:
Since this only works on linux, some kind of `assumeTrue` to skip the test on other platforms would be useful to prevent future "WTF" moments where the tests pass on macOS because they don't assert anything then fail in CI